fastlane/actions/synology_transfer.rb

```diff
@@ -89,6 +98,13 @@ def self.available_options
             description: "The root directory in Synology from which all artifacts will be placed",
             type: String
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :fallback_url,
+            description: "The fallback directory (local) in which artifacts will be placed in the event `destination_url` is unavailable",
+            type: String,
+            optional: true,
+            default_value: ""
```

Comment:
If this parameter is optional, should the default value be `null` or whatever the Ruby equivalent is?

Reply:
It could be, but it shouldn't matter in the end because we check if it's not empty before it gets used.